backend/app/api/endpoints/openapi_responses.py

```
@@ -747,33 +748,73 @@ def _normalize_protocol_type(
+                # pub/sub channel; the /internal/callback handler publishes events
+                pubsub_redis_client, pubsub_obj = (
+                    await session_manager.subscribe_callback_channel(
+                        assistant_subtask_id
+                    )
+                )
+                if pubsub_redis_client is None:
+                    raise RuntimeError("Failed to subscribe to callback stream channel")
+                # Fire-and-forget; executor sends events back via /internal/callback
+                asyncio.create_task(
+                    execution_dispatcher.dispatch(execution_request, emitter=None)
+                )
```

Ruff (0.15.12)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)